client/src/components/Jobs/Jobs.jsx

```diff
@@ -105,7 +106,7 @@ function JobsView({
           }}
           className="wb-link"
         >
-          View Details
+          {query.query_string ? <b>View Details</b> : 'View Details'}
```

Comment:
Should `HighlightSearchTerm` be used here also?

Reply:
No, since the "View Details" is a static text/button and `HighlightSearchTerm`is used to only highlight the query_string. In this case, we can just use <b>View Details</b> if query_string is present.